Runtimes/Overlay/Linux/glibc/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+add_subdirectory(clang)
+
+gyb_expand(Glibc.swift.gyb Glibc.swift)
+gyb_expand(tgmath.swift.gyb tgmath.swift)
+
+add_library(swiftGlibc
+  Glibc.swift
+  POSIXError.swift
+  Platform.swift
+  TiocConstants.swift
+  tgmath.swift)
+
+target_link_libraries(swiftGlibc
+  PRIVATE
+    swiftCore
+    SwiftGlibcClangOverlay)
+
+set_target_properties(swiftGlibc PROPERTIES
+  Swift_MODULE_NAME Glibc)
+
+target_compile_options(swiftGlibc PRIVATE
+  "SHELL:-Xcc -fno-omit-frame-pointer"
```

Comment:
Should we ensure these flags are only set when compiling for Swift? Or are we sure we will not be adding other non Swift files to this target?